src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -26,10 +85,21 @@ private string GetDebuggerDisplay()
         internal const int ListKind = 1;
 
         private readonly ushort _kind;
-        protected NodeFlags flags;
-        private byte _slotCount;
+        private NodeFlagsAndSlotCount _nodeFlagsAndSlotCount;
         private int _fullWidth;
 
+        protected NodeFlags flags
+        {
+            get => _nodeFlagsAndSlotCount.NodeFlags;
+            set => _nodeFlagsAndSlotCount.NodeFlags |= value;
+        }
+
+        private byte _slotCount
+        {
+            get => _nodeFlagsAndSlotCount.SlotCount;
+            set => _nodeFlagsAndSlotCount.SlotCount = value;
+        }
```

Comment:
bridge members to ensure nothing else needs to be updated.  It would be good to rename these in the future to actually be properties.